frontend/apps/reader/modules/readerfont.lua

```diff
@@ -209,11 +209,17 @@ end
 
 function ReaderFont:makeDefault(face)
     if face then
-        UIManager:show(ConfirmBox:new{
-            text = T( _("Set default font to %1?"), face),
-            ok_callback = function()
+        UIManager:show(MultiConfirmBox:new{
+            text = T( _([[Set %1 as default or fallback font? The fallback font ]] 
```

Comment:
Besides #1617, I only now noticed that this string concatenation doesn't seem to be picked up by Transifex. I'm guessing the problem is [here](https://github.com/koreader/koreader-misc/blob/4bae98523af9820424b0e382a708a9e6d4e5440d/gettext/lua_xgettext.py#L39-L41), but I think simply updating to [the latest from upstream](http://bazaar.launchpad.net/~widelands-dev/widelands/trunk/view/head:/utils/lua_xgettext.py) might solve the problem for us. I might investigate that tonight or this weekend.